src/components/TipInput.vue

```diff
@@ -10,8 +10,9 @@
       <img :src="iconTip">
       <AeAmountFiat
         v-if="!userAddress
-          && (!tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0)"
-        :amount="tipUrlStats.totalAmountAe"
+          && (tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0)"
```

Comment:
you have already implemented logic to calculate value/token below, why don't reuse it?
```
v-if="!userAddress && tipAmount.value"
```